CHANGELOG.md

```diff
@@ -7,6 +7,52 @@ Versions are `MAJOR.PATCH`.
 
 # Changelog
 
+Salt 3002.4 (2021-02-09)
```

Comment:
Not sure how this line `Salt 3002.4 (2021-02-09)` got here, but it should be deleted.

Reply:
Removed